4-WebApp-your-API/4-1-MyOrg/TodoListService/Controllers/TodoListController.cs

```diff
@@ -27,66 +31,161 @@ public TodoListController(IHttpContextAccessor contextAccessor)
             // Pre-populate with sample data
             if (TodoStore.Count == 0)
             {
-                TodoStore.Add(1, new Todo() { Id = 1, Owner = $"{_contextAccessor.HttpContext.User.Identity.Name}", Title = "Pick up groceries" });
-                TodoStore.Add(2, new Todo() { Id = 2, Owner = $"{_contextAccessor.HttpContext.User.Identity.Name}", Title = "Finish invoice report" });
+                TodoStore.Add(1, new Todo() { Id = 1, Owner = $"{GetObjectIdClaim(_contextAccessor.HttpContext.User)}", Title = "Pick up groceries" });
```

Comment:
Why not user HttpContext.User.GetObjectId()?

Reply:
Thanks. Changed.